contracts/Token.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.4;
+pragma solidity ^0.8.4;
```

Comment:
```suggestion
pragma solidity 0.8.4;
```
using the fixed solidity version was required by audit team, lets use 0.8.4 and keep all the same